sidebars.js

```diff
@@ -26,6 +26,18 @@ module.exports = {
         // "introduction/create",
       ],
     },
+    {
+      type: "category",
+      label: "User Guide",
+      collapsed: false,
```

Comment:
Let's keep collapsed `true`
and On left panel also lets call it homepage

Reply:
all changes done